### PR TITLE
add BIP 86 to derivation path

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

**Review by yahiheb — CHANGES_REQUESTED** [review]
Idk, this is confusing to me.
I guess currently we only use `m/84'/0'/0'` for change outputs in coinjoin, so I think we shouldn't say that Wasabi uses that as derivation path.

**Comment by lontivero** [comment]
Wasabi support of taproot is limited in the sense that we are currently not generating external taproot addresses. Technically specking  there is nothing preventing us to do it but for now we simply didn't want to touch the UI and introduce an address type selection.

**Review by yahiheb — APPROVED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 1]
```diff
@@ -474,7 +474,8 @@ The QR Code is displayed and can be saved as a png file by clicking on the Save
 
 ### What derivation paths does Wasabi use?
 
-Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki), so the main path is `m/84'/0'/0'`.
+Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki) (`m/84'/0'/0'`) and [BIP 86: Key Derivation for Single Key P2TR Outputs](https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki) (`m/86'/0'/0'`).
+Note that Taproot isn't supported yet so the main path is `m/84'/0'/0'`.
 
 On TestNet Wasabi uses the path `m/84'/1'/0'`.
 On RegTest it uses `m/84'/0'/0'`, and not the standard `m/84'/1'/0'` path.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:481]:
```suggestion
On TestNet Wasabi uses the paths `m/84'/1'/0'` and `m/86'/1'/0'`.
On RegTest it uses `m/84'/0'/0'` and `m/86'/0'/0'`, and not the standard `m/84'/1'/0'` and `m/86'/1'/0'` paths.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-UseWasabi.md [hunk 2]
```diff
@@ -474,10 +474,11 @@ The QR Code is displayed and can be saved as a png file by clicking on the Save
 
 ### What derivation paths does Wasabi use?
 
-Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki), so the main path is `m/84'/0'/0'`.
+Wasabi follows [BIP 84: Derivation scheme for P2WPKH Based Accounts](https://github.com/bitcoin/bips/blob/master/bip-0084.mediawiki) (`m/84'/0'/0'`) and [BIP 86: Key Derivation for Single Key P2TR Outputs](https://github.com/bitcoin/bips/blob/master/bip-0086.mediawiki) (`m/86'/0'/0'`).
+Note that Taproot isn't supported yet so the main path is `m/84'/0'/0'`.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-UseWasabi.md:478]:
```suggestion

Note that Wasabi is currently not generating external taproot addresses yet so the main path is `m/84'/0'/0'`.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Pull request merged**